ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -328,6 +334,8 @@
   :thread-pool            - custom thread pool instance for Jetty to use
   :truststore             - a truststore to use for SSL connections
   :trust-password         - the password to the truststore
+  :acceptor-threads       - the number of acceptor threads to use (default -1, i.e. defer to Jetty)
+  :selector-threads       - the number of selector threads to use (default -1, i.e. defer to Jetty)
```

Comment:
Lines go over 80 characters. Probably better to have:

```
  :acceptor-threads       - the number of acceptor threads to use
  :selector-threads       - the number of selector threads to use
```

If these options aren't set, I think a sensible default is implied.